depends/patches/zeromq/fix_mingw_link.patch

```diff
@@ -0,0 +1,35 @@
+commit 0977870006bd1b02a184608ccdd27152478e98ce
```

Comment:
Isn't an upstream commit? Can just drop the commit/author/date etc.

Reply:
Amended per your request.